pom.xml

```diff
@@ -1705,6 +1705,12 @@
         <scope>test</scope>
         <version>${mockito.jupiter.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-inline</artifactId>
+        <scope>test</scope>
+        <version>${mockito.jupiter.version}</version>
+      </dependency>
```

Comment:
Seems redundant as `mockito-inline` is already there?

Reply:
yeah, removed. but we still need to put it to hoodie-tests-common module.

Reply:
Makes sense.